Model/LdapUser.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace FR3D\LdapBundle\Model;
+
+use Symfony\Component\Security\Core\User\AdvancedUserInterface;
+
+/**
+ * Simple user object.
+ */
+class LdapUser implements LdapUserInterface, AdvancedUserInterface
```

Comment:
Good idea. As I can see the setEnable() is only needed in case of AdvancedUserInterface use. So this can be removed too. In my opinion we can move the setPassword('') to this if statement too or do you know something against that?